theurgy.lic

```diff
@@ -13,6 +13,8 @@ class TheurgyActions
   include DRCA
 
   def initialize(settings)
+    DRC.bput('stop play', /You stop/)
```

Comment:
be aware that bput handles stop play automatically. its pretty much the hack that this script thrives on.

Reply:
> Rather than stop I recommend pausing the script instead and then when theurgy is done then resume paused scripts.
> 
> Also, what if the instrument being played is held in hands, would stopping play or pausing the script still allow theurgy to work?

This won't actually solve the problem, as if you enter Theurgy while playing you'd need to unpause it, pausing the script is basically what play does internally, jut not a hard pause but logic based on currently running scripts. I didn't know bput handles the stop play as I've never experienced that working and had to manually add it to one of my helper scripts. I will check if I am just not updated, or if I am having unknown issues. I do know quite a few that have handled it outside of bput as well. 

Thanks for the feedback!